src/components/TodoList/TodoList.js

```diff
@@ -6,28 +6,78 @@ import {
   TodoContainer,
   StyledList,
   HeadContent,
+  StyleFilterPanel,
 } from "./TodoList.styled";
 
 function TodoList() {
-  const [todoInput, setTodoInput] = useState("");
-  const [list, setList] = useState([]);
+  const initialState = {
+    todoItem: [],
+    filter: "all",
+  };
+
+  const [state, setState] = useState(initialState);
+  const { todoItem, filter } = state;
+
+  const handleFilter = (status) => {
+    switch (status) {
+      case "all":
+        return todoItem;
+      case "checked":
+        let checkedItems = todoItem.filter((item) => item.status === "checked");
```

Comment:
nếu dùng const được thì nên dùng const hơn là dùng let Đức

https://stackoverflow.blog/2019/09/12/practical-ways-to-write-better-javascript/
mục "let and const"